src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -0,0 +1,88 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { BaAwaConfig } from './BaAwaConfig';
+import { BaAwaRules } from './BaAwaRules';
+import { MancalaTutorial } from '../common/MancalaTutorial';
+import { MancalaState } from '../common/MancalaState';
+import { MancalaDistribution, MancalaMove } from '../common/MancalaMove';
+import { TutorialStepFailure } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepFailure';
+
+const defaultConfig: MGPOptional<BaAwaConfig> = BaAwaRules.get().getDefaultRulesConfig();
+
+export class BaAwaTutorial extends Tutorial {
+
+    public gameName: string = $localize`Ba-awa`;
+
+    public tutorial: TutorialStep[] = [
+        MancalaTutorial.intro(
+            this.gameName,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        TutorialStep.informational(
+            $localize`Ba-awa`,
+            $localize`Bonus fact: Ba-awa, also called Adi, is played mainly in Ghana.`,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        MancalaTutorial.sowing(
+            new MancalaState([
+                [4, 4, 4, 4, 4, 4],
+                [4, 0, 4, 4, 4, 4],
+            ], 0, [0, 0]),
+        ),
+        TutorialStep.fromMove(
+            $localize`Multiple laps sowing`,
+            $localize`However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 2, 4, 0, 0, 0],
+                [0, 0, 0, 5, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`So, after landing in the house with 3 seeds, a second lap has been started`,
```

Comment:
la maison en avait deux mais en a 3 maintenant, c'est pas tant facile à formuler clairement ici; "after putting your last seed as the third in the house, a second lap has been started"?

Reply:
So, after landing in the house with 2 seeds (then 3), a second lap has been started
C'pas mieux ? Là tu part en tournure de phrase et touuut looo

Reply:
2 seeds (3 after drop) ?